collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadList.java

```diff
@@ -78,6 +78,7 @@
 
     private static final int PROGRESS_DIALOG = 1;
     private static final int AUTH_DIALOG = 2;
+    private static final int CANCEL_DIALOG = 3;
```

Comment:
The purpose of the dialog is to block and show that a cancellation is in progress. A better name might be `CANCELLATION_DIALOG`. `CANCEL_DIALOG` suggests to me that the user might be able to initiate some sort of cancellation.